src/v2/blobs.rs

```diff
@@ -1,11 +1,74 @@
 use futures::Stream;
 use reqwest;
 use reqwest::StatusCode;
+use sha2::{self, Digest};
 use v2::*;
 
 /// Convenience alias for future binary blob.
 pub type FutureBlob = Box<futures::Future<Item = Vec<u8>, Error = Error>>;
 
+#[derive(Clone, Debug, PartialEq)]
+pub struct LayerDigest {
```

Comment:
Name bikeshedding: I think this isn't specific to layers, according to spec other objects (e.g. manifests) may return a `Docker-Content-Digest`. Spec calls this "content digest", I'd suggest to stick to that.

Reply:
I was going to have a discussion around this. I would agree that this can be used elsewhere and I want to find a reusable place for this in the internal API.

Reply:
Moved and renamed into a separate module